src/components/comparison/comparison-side/comparison-side.tsx

```diff
@@ -205,8 +208,13 @@ export const ComparisonSide = ({
     if (compareButtonMode === CompareButtonMode.Comparing) {
```

Comment:
Looks like we don't need to check compare button here having special boolean prop.